### PR TITLE
Add config for disk IO limits.

### PR DESCRIPTION
**Pull request by yhaliaw**
- Add config for disk read IO limit and disk write IO limit.
- Bug fix for check-runner, fix the unit test for it as well. The bug was in both the code and unit test.

Pending discussion with SRE on the disk IO read and write limit.
The placeholder value 100 MiB read limit and 50 MiB write limit is used for now.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -40,6 +40,18 @@ options:
     description: >
       Amount of disk space to allocate to root disk for virtual machine runner. Positive integers
       with GiB suffix.
+  vm-disk-read:
+    type: string
+    default: 100MiB
+    description: >
+      Limit of disk read IO per second of the root disk for virtual machine runner. Positive integers
```
Comment (arturo-seijas) [inline config.yaml:47]: Why MiB specifically? I don't see that limitation in the code. Also, the name of the config doesn't seem too obvious to me, I'd rename it to `vm-disk-read-limit` or something similar (same for the other config).

Comment (yhaliaw) [reply config.yaml:48]: I have updated the descriptions.

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Comment by yhaliaw** [comment]
The default installation of LXD no longer support disk IO limit. The setting does not work anymore.
Close this PR due to this.

**Pull request closed without merging**